packages/api/src/get-repo.js

```diff
@@ -20,15 +20,35 @@ const { InvalidParameterError, handleError } = require('../lib/errors');
 const { isJson } = require('../util/validation');
 const FILTERS_ALLOWED = ['tag', 'ref', 'os', 'limit', 'matrix'];
 
-class GetBuildHandler {
+class GetRepoHandler {
   constructor (app, client) {
     this.app = app;
     this.client = client;
   }
 
   listen () {
-    // return the recent builds based on parameters
     this.app.get('/api/repo/:orgname/:reponame', async (req, res, next) => {
+      try {
+        const repoid = firebaseEncode(req.params.orgname + '/' + req.params.reponame);
+
+        const d = new Date();
+        const metadata = await this.client.collection(global.headCollection).doc(repoid).get();
+        const d2 = new Date();
+        const totalTime = d2.getTime() - d.getTime();
+        console.log('this took ' + totalTime);
```

Comment:
I would make these timing messages a bit more descriptive, for the benefit of our server logs, one option perhaps  look at using middleware like [response-time](http://expressjs.com/en/resources/middleware/response-time.html) and we'd get timings for all requests.

Reply:
deleted. Left over from testing.